syntax/swift.vim

```diff
@@ -142,6 +145,8 @@ syntax keyword swiftAttributes
       \ @UIApplicationMain
       \ @warn_unused_result
 
+syntax keyword swiftConditionStatement #available
```

Comment:
Well the Swift book calls them a `BuildConfigurationStatement`, which `#available` still isn't.

We could call it `HashtagStatements` or something :P.

Any thoughts on what we could call the group?